### ISSUE TITLE
Use conflicts from 3-way diff to determine if a method has moved

### ISSUE THREAD

**Issue by haroldhues** [semdiffdotnet/semdiff#8]

**Comment by haroldhues** [comment]
To do this there needs to be a way to see that a method has been removed. I have experimented with this in tests, but have come up with issues. When there is lots of whitespace at the ends of the method that changed then the node that surrounds it best may be the class and not the method. 

This is the general procedure I am thinking about:
  1. Split the conflict at the edges of the methods (that way if two methods are removed, there are two changes).
  2. Get the node that contains the change, if it is a method declaration syntax continue
  3. Check that the change is equal to the method declaration syntax's size

Also consider this troubling case that may occur, I will need to see if it can even be reproduced first though... In this situation, the bracket follows the wrong function...
```
        public void Function12(User user)
        {
            if (string.IsNullOrWhiteSpace(user.eyeColor))
                throw new InvalidOperationException();
<<<<<<< original
        }
        public void Function3(User user)
        {
            if (string.IsNullOrWhiteSpace(user.company))
                throw new InvalidOperationException();
            Function4(user);
=======
>>>>>>> new
        }
```
